modules/media/php/mediafileprovisioner.class.inc

```diff
@@ -38,22 +38,22 @@ class MediaFileProvisioner extends \LORIS\Data\Provisioners\DBRowProvisioner
     {
         parent::__construct(
             "SELECT m.file_name as fileName,
-                    c.PSCID as pscid,
```

Comment:
This shouldn't be making whitespace changes

Reply:
I know but ended up with missalign lines when I added a line here (that was later removed). I had to replace lots of tab with space.